sample-dcrwallet.conf

```diff
@@ -47,19 +47,59 @@
 ; RPC client settings
 ; ------------------------------------------------------------------------------
 
+; The server and port used for dcrd websocket connections.
+; rpcconnect=localhost:9109
+
+; File containing root certificates to authenticate a TLS connections with dcrd
+; cafile=~/.dcrdwallet/dcrd.cert
+
+
+
+; ------------------------------------------------------------------------------
+; Proxy/Tor settings
+; ------------------------------------------------------------------------------
```

Comment:
You should probably add a blank line here so that this header looks similar to the other ones.